npm/private/npm_package.bzl

```diff
@@ -459,6 +480,35 @@ def npm_package(
         **kwargs
     )
 
+def _publish_target(name, package, args = [], visibility = None, tags = [], testonly = False):
+    # Always tag the target manual since we should only build it when the final target is built.
+    tags = tags + ["manual"]
+
+    expand_template(
+        name = "{}_mjs".format(name),
+        template = "@aspect_rules_js//npm/private:npm_package.mjs",
+        out = "{}.mjs".format(name),
+        substitutions = {
+            "{{PACKAGE_DIR}}": "$(rlocationpaths :{})".format(package),
```

Comment:
Do you know what version of bazel `rlocationpaths` was added it. Did it exist in Bazel 5?

Reply:
Thanks for checking. Technically this would be a breaking change for anymore not yet updated to 5.4.0 and using npm_package so I think we should implement without relying `rlocations`. We could leave a TODO for rules_js 2.0 and switch to rlocations at that point.

Reply:
rootpath is also okay.